lib/rules/declaration-block-no-shorthand-property-overrides/index.mjs

```diff
@@ -37,9 +37,9 @@ const rule = (primary) => {
 					longhandSubPropertiesOfShorthandProperties
 				).get(unprefixedProp);
 
-				if (!overrideables) {
-					declarations.set(prop.toLowerCase(), prop);
```

Comment:
question
medium

Shouldn't this being in this `if` block have produced a bug?
i.e. a test is probably missing to cover that case or the setter was useless

Reply:
I feel like this logic change makes sense because test cases other than `border-block*` properties pass.

Reply:
If I understand the change, a new passing test is needed or it needs to be kept in the `if`.
I gotta checkout your branch to check.